modules/ROOT/pages/migration-munit-mock-processor-changes.adoc

```diff
@@ -0,0 +1,71 @@
+= Migrating the Mock Processor (MUnit)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+. The name of the processor has changed from when to mock-when.
+. The attribute messageProcessor has change to processor.
+. In the child node with attribute the attribute name has changed to attributeName.
+. In the child node then-return the attribute payload must now be a child node.
+. The child node invocation-properties has changed variables.
```

Comment:
The child node `invocation-properties` has changed **to** `variables`. (confirm if this is correct)

Reply:
done via suggestion

Reply:
verified change